cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/dns/HttpContactPointBootstrap.scala

```diff
@@ -67,7 +63,7 @@ private[dns] class HttpContactPointBootstrap(
         s"Uri: $baseUri, this node's remoting address: ${cluster.selfAddress}")
   }
 
-  private implicit val mat = ActorMaterializer()
+  private implicit val mat = ActorMaterializer()(context.system)
```

Comment:
btw, do we check somewhere if `akka.cluster.seed-nodes` config is non-empty? If non-empty I think we should not start joining in the bootstrap extension.

Reply:
Oh good point, thanks! Will add that and a test

Reply:
This is done